src/main/solution_space/measured_vertex/measured_vertex_v_1.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+from __future__ import annotations
```

Comment:
Please, add an empty line between **Copyright** and **from future import annotations**